modules/gapi/src/api/kernels_imgproc.cpp

```diff
@@ -97,6 +97,22 @@ GMat Canny(const GMat& src, double thr1, double thr2, int apertureSize, bool l2g
     return imgproc::GCanny::on(src, thr1, thr2, apertureSize, l2gradient);
 }
 
+std::tuple<GArray<Point2f>, GArray<uchar>, GArray<float>> calcOpticalFlowPyrLK(const GMat& prevImg,
```

Comment:
All stuff you've added here should be moved to the separate file which is dedicated for definition of G-API operations wrappers for OpenCV video module functions:  modules/gapi/src/api/kernels_video.cpp (which you should create).